Recruitment/JobSpecs/2_why_you_platform_engineer.md

```diff
@@ -5,22 +5,22 @@
 * Explore up-and-coming technologies and software products. Read; experiment; learn; teach.
 
 ### Skills and experience we’re after:
-* Cloud
+* Hands-on, best-of-breed software engineering, within autonomous, cross-functional delivery teams
+* Experience in at least one ubiquitous programming language and polyglot tendencies
+* Cloud, \*aaS
 * CI/CD
-* \*aaS
 * Monitoring
 * Analytics
-* Automation
+* Comfortable with technical implementation to high-level architecture.
 
 ### We currently use, but aren’t wedded to:
+* Java, Spring MVC, SpringBoot
```

Comment:
Maybe add Node.js?  We've been doing that on the server side w/ our products & accelerators